package.json

```diff
@@ -14,21 +17,37 @@
   "main": "app/background.js",
   "dependencies": {
     "@kubernetes/client-node": "^0.13.0",
+    "bufferutil": "^4.0.3",
+    "canvas": "^2.6.1",
     "core-js": "^3.8.1",
     "deepmerge": "^4.2.2",
+    "jquery": "^3.5.1",
+    "jsonpath": "^1.0.2",
     "lodash": "^4.17.20",
+    "sass": "^1.32.2",
     "semver": "^7.3.4",
     "sudo-prompt": "^9.2.1",
+    "utf-8-validate": "^5.0.4",
+    "vue": "^2.6.12",
+    "vue-router": "^3.4.9",
```

Comment:
We shouldn't need this anymore, but that can be a follow-up.

Reply:
Are you talking about utf-8-validate, or all three lines?

Reply:
I'm talking about just `vue-router` here, though I guess `vue` too.

`utf-8-validate`, from previous experience, was pulled in because of warnings.